services/movie_session.py

```diff
@@ -0,0 +1,62 @@
+from db.models import MovieSession
+from db.models import CinemaHall
+from db.models import Movie
+
+
+def create_movie_session(
+        movie_show_time: str,
+        movie_id: int,
+        cinema_hall_id: int
+) -> None:
+    MovieSession.objects.create(
+        show_time=movie_show_time,
+        movie=Movie.objects.get(id=movie_id),
+        cinema_hall=CinemaHall.objects.get(id=cinema_hall_id),
```

Comment:
These two queries are not necessary: you can set foreign key directly by id: `movie_id`.